src/CliCore.ts

```diff
@@ -27,6 +27,7 @@ export default class CliCore {
         this.templateProcessor = null;
         this.logLevel = "info";
     }
+    public onInit;
```

Comment:
This needs to be properly typed as a function from typescript OV. At present this would allow someone to set string 'foo' for onInit.